collections/Laud_Misc/MS_Laud_Misc_306.xml

```diff
@@ -83,35 +133,299 @@
                <msPart xml:id="MS_Laud_Misc_306-part2">
                   <msIdentifier>
                      <altIdentifier type="partial">
-                        <idno type="part">MS. Laud Misc. 306 – Part 2</idno>
+                        <idno type="part">MS. Laud Misc. 306, fols. 2–115</idno>
```

Comment:
I appreciate the difficulties with "part", but I think something as well as a folio range is useful; how about `fols. 2-115 (codicological unit I/A/ whatever)` (assuming that fol. 1 above is an endleaf not a codicological unit (?)